tests/queries/0_stateless/02115_insert_deduplication_token.sql

```diff
@@ -0,0 +1,20 @@
+-- insert data duplicates by providing deduplication token on insert
+
+DROP TABLE IF EXISTS replicated_with_duplicates SYNC;
+
+CREATE TABLE IF NOT EXISTS replicated_with_duplicates (
+    id Int32, val UInt32
+) ENGINE=ReplicatedMergeTree('/clickhouse/tables/{database}/test_02115/replicated_with_duplicates', '{replica}') ORDER BY id;
+
+-- only one row will be inserted
+INSERT INTO replicated_with_duplicates VALUES(1, 1001);
+INSERT INTO replicated_with_duplicates VALUES(1, 1001);
+
+SELECT * FROM replicated_with_duplicates;
+
+select 'insert duplicate by providing deduplication token';
+set insert_deduplication_token = '1';
+INSERT INTO replicated_with_duplicates VALUES(1, 1001);
```

Comment:
Ok, let's also check that the data will be deduplicated by `insert_deduplication_token`.

Reply:
And just in case - let's create two replicas and do inserts on different tables.